src/Raven.Server/Documents/Replication/LiveReplicationPerformanceCollector.cs

```diff
@@ -168,8 +167,7 @@ private void OutgoingHandlerAdded(OutgoingReplicationHandler handler)
 
         private void OutgoingDocumentsSend(OutgoingReplicationHandler handler)
         {
-            ReplicationHandlerAndPerformanceStatsList<OutgoingReplicationHandler, OutgoingReplicationStatsAggregator> stats;
-            if (_outgoing.TryGetValue(handler, out stats) == false)
+            if (_outgoing.TryGetValue(handler, out ReplicationHandlerAndPerformanceStatsList<OutgoingReplicationHandler, OutgoingReplicationStatsAggregator> stats) == false)
```

Comment:
I was just consistent with not using `var` for out params, but yes, in this case it make sense to use it.